dev/com.ibm.ws.jpa.container.core/resources/com/ibm/ws/jpa/jpa.nlsprops

```diff
@@ -302,6 +302,10 @@ NOT_COMPLIANT_WITH_JPA21_CWWJP0054E=CWWJP0054E: The {0} implementation, {1}, is
 NOT_COMPLIANT_WITH_JPA21_CWWJP0054E.explanation=The JPA persistence provider cannot be used because it lacks implementation of methods that are required by the JPA 2.1 specification.
 NOT_COMPLIANT_WITH_JPA21_CWWJP0054E.useraction=Obtain a newer version of the JPA persistence provider that complies with the JPA 2.1 specification.
 
+ERROR_TRANSFORMING_CLASS_CWWJP0055E=CWWJP0055E: An error occurred while attempting to transform the Java Class {0}.  The following failure detail has been gathered: {1}
+ERROR_TRANSFORMING_CLASS_CWWJP0055E.explanation=An unexpected error occurred while transforming the class, which is usually attributable to invalid Java bytecode.
+ERROR_TRANSFORMING_CLASS_CWWJP0055E.useraction=See message for details.
```

Comment:
Will the message really provide some action for the user to perform?  Something in the detail?  Could we recommend clearing the class cache?  Anything else we've ever seen resolve this type of issue?

Reply:
> Will the message really provide some action for the user to perform?

There are a couple of actions that may be suitable.  Clearing the class cache for one.  But it is likely that the application archive had been packaged improperly.

Reply:
Pushed the change in an update commit.